packages/coil-extension/src/content/services/ContentScript.ts

```diff
@@ -161,13 +161,14 @@ export class ContentScript {
           this.onFrameAllowedChanged(request)
         } else if (request.command === 'tip') {
           debug('sendTip event')
-          const detail: TipEvent['detail'] = {
+          const detail: MonetizationProgressEvent['detail'] = {
             amount: request.data.amount,
             assetCode: request.data.assetCode,
             assetScale: request.data.assetScale,
-            paymentPointer: request.data.paymentPointer
+            paymentPointer: request.data.paymentPointer,
```

Comment:
Backend not yet sending receipts ?

Reply:
We are not passing them through yet. I'm tackling that next.